src/main/java/org/apache/sysds/runtime/instructions/SPInstructionParser.java

```diff
@@ -126,7 +126,9 @@ public class SPInstructionParser extends InstructionParser
 		String2SPInstructionType.put( "uac*"    , SPType.AggregateUnary);
 		String2SPInstructionType.put( "uatrace" , SPType.AggregateUnary);
 		String2SPInstructionType.put( "uaktrace", SPType.AggregateUnary);
-		String2SPInstructionType.put( "uacdap"  , SPType.AggregateUnary);
+		String2SPInstructionType.put( "uacd"    , SPType.AggregateUnary);
+		String2SPInstructionType.put( "uacdr"   , SPType.AggregateUnary);
+		String2SPInstructionType.put( "uacdc"   , SPType.AggregateUnary);
```

Comment:
same here, should we add the approximate methods as well?

Reply:
They are already there- they are in a separate "section" for sketch operators:
```
		String2SPInstructionType.put( "uacd"    , SPType.AggregateUnary);
		String2SPInstructionType.put( "uacdr"   , SPType.AggregateUnary);
		String2SPInstructionType.put( "uacdc"   , SPType.AggregateUnary);

		// Aggregate unary sketch operators
		String2SPInstructionType.put( "uacdap" , SPType.AggregateUnarySketch);
		String2SPInstructionType.put( "uacdapr", SPType.AggregateUnarySketch);
		String2SPInstructionType.put( "uacdapc", SPType.AggregateUnarySketch);
```